sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -25,11 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Update="ShimLayer\*.cs">
-      <SonarQubeExclude>true</SonarQubeExclude>
-    </Compile>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />

```

Comment:
I think this was a rebasing mistake, there should not be any reference to `Newtonsoft.Json` on master anymore